### PR TITLE
Fix document about pi in sql

### PR DESCRIPTION
**Pull request by bananaaggle**
Fix PI in document. I test PI in sql, and it works. In source code, this `SqlFunction` is bound to its implementation. But I can not find it in document. I think it should be added in document.

<hr>

This PR has:
- [ ] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: docs/querying/sql.md [hunk 1]
```diff
@@ -376,6 +376,7 @@ to FLOAT. At runtime, Druid will widen 32-bit floats to 64-bit for most expressi
 
 |Function|Notes|
 |--------|-----|
+|`PI`|Constant Pi.|
```
Comment (asdf2014) [inline docs/querying/sql.md:379]:
```suggestion
|`PI()`|Constant Pi.|
```

**Review by asdf2014 — CHANGES_REQUESTED** [review]
Nice catch :+1:

**Review by AshishKapoor — APPROVED** [review]

File: docs/querying/sql.md (hunk 1 above)
Comment (bananaaggle) [reply docs/querying/sql.md:379]: Just like `CURRENT_TIMESTAMP`, this function not need a `()`. Actually, sql will not work when write PI as `PI()`.

**Review by zhangyue19921010 — APPROVED** [review]
LGTM

**Review by asdf2014 — APPROVED** [review]

**Pull request merged**